readme.md

```diff
@@ -8,6 +8,8 @@
 > Terminal string styling done right
 
 [![Build Status](https://travis-ci.org/sindresorhus/chalk.svg?branch=master)](https://travis-ci.org/sindresorhus/chalk) [![](http://img.shields.io/badge/unicorn-approved-ff69b4.svg)](https://www.youtube.com/watch?v=9auOCbH5Ns4)
+[![Coverage Status](https://coveralls.io/repos/sindresorhus/chalk/badge.svg?branch=)](https://coveralls.io/r/sindresorhus/chalk?branch=)
```

Comment:
I think I might have cargo culted this badge from an old version of the coveralls.io UI, which generated the badge in this way. I'll make sure to use `branch=master` as I instrument other codebases.